frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -24,21 +25,45 @@ import { trendsLogic } from 'scenes/trends/trendsLogic'
 import { FunnelStepReference } from 'scenes/insights/InsightTabs/FunnelTab/FunnelStepReferencePicker'
 import { eventDefinitionsModel } from '~/models/eventDefinitionsModel'
 
+function aggregateBreakdownResult({ result: breakdownList, ...apiResponse }: FunnelResultWithBreakdown): FunnelResult {
+    let result: FunnelStep[] = []
+    if (breakdownList.length) {
+        result = breakdownList[0].map((step, i) => ({
+            ...step,
+            breakdown_value: step.breakdown as any,
+            count: breakdownList.reduce((total, breakdownSteps) => (total += breakdownSteps[i].count), 0),
+            breakdown: breakdownList.reduce((allEntries, breakdownSteps) => [...allEntries, breakdownSteps[i]], []),
+            average_conversion_time: null, // TODO API needs to return a weighted average.
+            people: [],
```

Comment:
We could concatenate and de-dupe the UUIDs from breakdown values here, but it sounds expensive and Li's person modal fetches the data again anyway.